lib/App/index.js

```diff
@@ -152,8 +139,10 @@ class App {
           throw new Error(`Forbidden permission: ${permission}`);
         }
 
-        if (permission === 'homey:manager:api' && levelPublish) {
-          console.warn('Warning: using the homey:manager:api permission will require a more thorough review. It may take longer than usual to review your app.');
+        if (permission === 'homey:manager:api') {
+          if (level >= VALIDATION_LEVELS.PUBLISH) {
+            console.warn('Warning: using the homey:manager:api permission will require a more thorough review. It may take longer than usual to review your app.');
```

Comment:
Maybe this should use the `Log` method?

Reply:
Maybe, don't know if `node-homey-lib` as such a method. Code was already like this so seems fine to keep it like this.

Reply:
It does, might be more noticible using colors, but not an issue really: https://github.com/athombv/node-homey/search?q=Log